### ISSUE TITLE
Floating fence in Booty Bay

### ISSUE THREAD

**Issue by Tenyar97** [slowtorta/turtlewow-bug-tracker#2477]
Next to Dizzy One-Eye in Booty Bay is a fence post rail not set to the catwalk. It cuts through the top leaving two posts and a tiki torch free-standing.
![WoW_tweaked 2022-11-16 21-33-56](https://user-images.githubusercontent.com/3698491/202590050-bf9c853a-c16a-4e89-bfad-894999703753.png)

**Comment by redmagejoe** [comment]
This issue has actually been in every server I've played after vanilla retail, and I can't remember if I saw this back in 2004. Having said that, it's led me to assume this was an error made by the actual Blizzard map designers. If Turtle could fix it, that would be great, but it depends on if Booty Bay is a single WMO or not, as the fix might be more complex than lowering a single world object. Let's hope that's not the case.
